lib/nrf_modem_lib/lte_connectivity/lte_connectivity.c

```diff
@@ -64,6 +70,99 @@ void nrf_modem_fault_handler(struct nrf_modem_fault_info *fault_info)
 	fatal_error_notify_and_disconnect();
 }
 
+static void become_active(void)
```

Comment:
Some comments for these functions would be nice, this and `become_dormant` actually mean in this context.